packages/template-retail-react-app/app/constants.js

```diff
@@ -201,3 +201,33 @@ export const STORE_LOCATOR_DISTANCE = 100
 export const STORE_LOCATOR_NUM_STORES_PER_LOAD = 10
 export const STORE_LOCATOR_DISTANCE_UNIT = 'km'
 export const STORE_LOCATOR_IS_ENABLED = true
+
+// Constants for shopper context
+export const SHOPPER_CONTEXT_FIELD_TYPES = {
+    INT: 'int',
+    DOUBLE: 'double',
+    ARRAY: 'array'
+}
```

Comment:
Perhaps comment that these are for _non-string_ types. You only need to identify the non-string types in the `SHOPPER_CONTEXT_SEARCH_PARAMS` below.